### PR TITLE
PMM-7 fix noctx linter warnings

### PR DESCRIPTION
**Pull request by atymchuk**
PMM-7

- [ ] https://github.com/percona/pmm/issues/1541

This PR fixes the `noctx` rule, which was previously disabled

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ body:
     attributes:
       label: Relevant logs
       description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
-      render: shell
+      render: Shell
```
Comment (atymchuk) [inline .github/ISSUE_TEMPLATE/bug.yml:51]: Suggested by the linter

File: .golangci.yml [hunk 2]
```diff
@@ -139,12 +139,12 @@ issues:
   exclude-use-default: false
   exclude:
     # gas: Duplicated errcheck checks
-    - 'G104: Errors unhandled'
+    - "G104: Errors unhandled"
```
Comment (atymchuk) [inline .golangci.yml:142]: Those are autofixes, since quotes were inconsistent

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (artemgavrilov) [inline .github/ISSUE_TEMPLATE/bug.yml:51]: GH uses lowercase in their examples: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Review by artemgavrilov — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (atymchuk) [reply .github/ISSUE_TEMPLATE/bug.yml:51]: Shame on them - it doesn't pass their own schema validation 😄 
https://json.schemastore.org/github-issue-forms.json

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**